beef/beef_tx_test.go

```diff
@@ -51,51 +54,51 @@ func TestDecodeBEEF_DecodeBEEF_HappyPaths(t *testing.T) {
 				},
 				Transactions: []*TxData{
 					{
-						Transaction: &bt.Tx{
+						Transaction: &trx.Transaction{
 							Version:  1,
 							LockTime: 0,
-							Inputs: []*bt.Input{
+							Inputs: []*trx.TransactionInput{
 								{
-									PreviousTxSatoshis: 0,
-									PreviousTxOutIndex: 1,
-									SequenceNumber:     4294967295,
-									PreviousTxScript:   nil,
+									// PreviousTxSatoshis: 0,
+									// PreviousTxOutIndex: 1,
+									SequenceNumber: 4294967295,
+									// PreviousTxScript:   nil,
 								},
 							},
-							Outputs: []*bt.Output{
+							Outputs: []*trx.TransactionOutput{
 								{
 									Satoshis:      26174,
-									LockingScript: bscript.NewFromBytes([]byte("76a9146bfd5c7fbe21529d45803dbcf0c87dd3c71efbc288ac")),
+									LockingScript: script.NewFromBytes([]byte("76a9146bfd5c7fbe21529d45803dbcf0c87dd3c71efbc288ac")),
 								},
 							},
 						},
-						BumpIndex: func(v bt.VarInt) *bt.VarInt { return &v }(0x0),
+						BumpIndex: func(v trx.VarInt) *trx.VarInt { return &v }(0x0),
 					},
 
 					{
-						Transaction: &bt.Tx{
+						Transaction: &trx.Transaction{
 							Version:  1,
 							LockTime: 0,
-							Inputs: []*bt.Input{
+							Inputs: []*trx.TransactionInput{
 								{
-									PreviousTxSatoshis: 0,
-									PreviousTxOutIndex: 0,
-									SequenceNumber:     4294967295,
-									PreviousTxScript:   nil,
+									// PreviousTxSatoshis: 0,
+									// PreviousTxOutIndex: 0,
+									SequenceNumber: 4294967295,
+									// PreviousTxScript:   nil,
```

Comment:
Do we have a jira task for that?

Reply:
Sorry, didn't see those comments -> I took this PR as draft. 
I removed comments and moved to equivalent fields from go-sdk structures.